src/app/pages/send-tokens/components/recipient-field/recipient-field.tsx

```diff
@@ -79,6 +91,9 @@ function RecipientFieldBase(props: RecipientField) {
           data-testid={SendFormSelectors.InputRecipientField}
         />
       </InputGroup>
+      <Caption cursor="pointer" onClick={showAccountsDrawer}>
+        Transfer between my account
+      </Caption>
```

Comment:
This isn't accessible html. It can't be selected by keyboard/screen reader.

Could you make it a button styled as a caption? I think you can add `<Caption as="button"...` to do this, then the manual cursor pointer isn't needed.